domain/src/main/java/io/homeassistant/companion/android/domain/integration/EntityResponse.kt

```diff
@@ -0,0 +1,12 @@
+package io.homeassistant.companion.android.domain.integration
+
+import java.util.Calendar
+
+data class EntityResponse(
+    val entityId: String,
+    val state: String,
+    val attributes: Map<String, String>,
```

Comment:
This is `Map<String, Any>`

Reply:
If it helps, in TypeScript we have a base entity and then `ZoneEntity`, `LightEntity` etc where we fill in the attributes with the types we expect.